cfme/tests/test_replication.py

```diff
@@ -170,23 +199,10 @@ def test_replication_appliance_add_multi_subscription():
         casecomponent: Replication
         initialEstimate: 1/4h
         startsin: 5.7
-    """
-    pass
-
-
-@pytest.mark.manual
-@test_requirements.replication
-@pytest.mark.tier(1)
-def test_replication_appliance_set_type_remote_ui():
```

Comment:
This test is quite redundant as the its steps are included in every test needing the replication working.